x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/cohere/CohereService.java

```diff
@@ -166,24 +166,14 @@ private static CohereModel createModel(
         return switch (taskType) {
             case TEXT_EMBEDDING -> new CohereEmbeddingsModel(
                 inferenceEntityId,
-                taskType,
```

Comment:
Just removing the task type and service name parameters as both of these are known from model type.